shared/general/config-variables-pi

```diff
@@ -0,0 +1,6 @@
+Name | Default | Description
+--- | --- | ---
+RESIN_HOST_CONFIG_disable_splash | 1 | Enable / Disable the {{ $names.company.lower }} splash screen
```

Comment:
There's another file in this PR where it says about `disable_splash` ... "This setting disables the Raspberry Pi rainbow splash screen but does not disable the {{ $names.company.lower }} logo splash screen." - seems to be some contradiction here?

Reply:
Great catch, this was pulled in from another part of the docs and not updated (this was also recently clarified in the dashboard).